CMakeSettings.json

```diff
@@ -1,5 +1,5 @@
 ﻿{
-  "environments": [ { "BuildNumber": "5180" } ],
+  "environments": [ { "BuildNumber": "5181" } ],
```

Comment:
this one is my fault, I released without incrementing that buildNumber

you are correct, it should have been updated